code/modules/unit_tests/unit_test.dm

```diff
@@ -143,11 +142,20 @@ GLOBAL_LIST_EMPTY(unit_test_mapping_logs)
 		test.log_for_test(text, "error", file, line)
 
 		// Normal log message
-		log_entry += "\tREASON #[reasonID]: [text] at [file]:[line]"
+		log_entry += "\tFAILURE #[reasonID]: [text] at [file]:[line]"
 
 	var/message = log_entry.Join("\n")
 	log_test(message)
 
```

Comment:
I'm not a big fan of having to click the dropdown just to see that the test passed. Would be optimal if the dropdown only show up if the test had output messages or failed imo.

Reply:
Especially since tests aren't sorted alphabetically or anything

Reply:
The structure in order to do that is way more involved than you might think, since *any* log, *any* runtime, etc is put into stdout (and I'm not confident about if we can even control runtime output). You know a test passes if it doesn't fail, more or less. Here's how it looked before for comparison:

![image](https://user-images.githubusercontent.com/35135081/194440619-cf8a79a6-88c9-4ad9-8be2-fe7f0052d096.png)